spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -82,13 +99,19 @@ public void testRepository() throws InterruptedException {
 
 		List<TestEntity> foundByCustomQuery = Collections.emptyList();
 		for (int i = 0; i < QUERY_WAIT_ATTEMPTS; i++) {
-			if (!foundByCustomQuery.isEmpty()) {
+			if (!foundByCustomQuery.isEmpty() && this.testEntityRepository
+					.countByShapeAndColor("round", "red") == 3) {
```

Comment:
didn't we decide that we are not going to support `count`?

Reply:
My bad, I missed this one.